letsencrypt/cli.py

```diff
@@ -1583,6 +1584,16 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):
         "security", "--rsa-key-size", type=int, metavar="N",
         default=flag_default("rsa_key_size"), help=config_help("rsa_key_size"))
     helpful.add(
+        "security", "--ecdsa-curve", default=flag_default("ecdsa_curve"),
+        help="Set the certificate ECDSA curve. Current possible curves: "
+        "prime256v1 (default) or secp384r1")
```

Comment:
Let's use the more easily remembered [FIPS 186-4](http://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.186-4.pdf) names for these curves: P-256 and P-384. Also, please extract them to a constant and use it in both a validator for the flag and in obtain_certificate.

Also, should this be `config_help("ecdsa_curve")`?